store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -8156,6 +8159,35 @@ public void modifyContact(OperationContext octxt, int contactId, ParsedContact p
         return result;
     }
 
+    /**
+     * Returns a list of contacts don't exist in any contacts folders.
+     * @param octxt
+     * @param listOfContactsToBeModified This list consist list of Map, email as key and Pair of ContactId and 
+     * DisplayName as Value
+     * @return nothing
+     */
+    public void modifyAutoContact(OperationContext octxt, List<Map<String, Pair<Integer, String>>> listOfContactsToBeModified)
+            throws IOException {
+        String email = null;
+        for(Map<String, Pair<Integer, String>> map: listOfContactsToBeModified) {
```

Comment:
* add a space: `for (Map`
* add a space: `map : listOfContactsToBeModified`

Reply:
done